test/Microsoft.ML.Benchmarks/Text/MultiClassClassification.cs

```diff
@@ -74,26 +50,15 @@ public void CV_Multiclass_WikiDetox_BigramsAndTrichar_LightGBMMulticlass()
                     " xf=Convert{col=logged_in type=R4}" +
                     " xf=CategoricalTransform{col=ns}" +
                     " xf=TextTransform{col=FeaturesText:comment wordExtractor=NGramExtractorTransform{ngram=2}}" +
-                    " xf=Concat{col=Features:FeaturesText,logged_in,ns} tr=LightGBMMulticlass{}";
+                    " xf=Concat{col=Features:FeaturesText,logged_in,ns}" +
+                    "tr=LightGBMMulticlass{iter=10}";
```

Comment:
did you miss a space here? There used to be a space between `} tr`. Now it appears the previous line ends with no space, and this line begins with no space.